COMPATIBILITY.md

```diff
@@ -0,0 +1,65 @@
+# Compatibility table for services
+
+Methods/properties not listed in here are implemented for all services.
+In case of an error, please report with issue.
+
+## Comments
+
+### `IssueComment`
+
+|               | GitHub | GitLab | Pagure |
+| ------------- | :----: | :----: | :----: |
+| `body`-setter |   ✔    |   ✔    |   ✘    |
```

Comment:
It is also getter, isn't it? It's not clear what the cross means. Does it mean that Pagure implementation does not have `body` at all?

What about two lines or `get`/`set` in the parentheses?